packages/adders/lucia/index.ts

```diff
@@ -300,14 +294,47 @@ export default defineAdder({
 	
 							return { session, user };
 						}`;
-					ms.append(`\n\n${validateSession}`);
+					ms.append(`\n\n${validateSessionToken}`);
 				}
 				if (typescript && !ms.original.includes('export type SessionValidationResult')) {
 					const sessionType =
-						'export type SessionValidationResult = Awaited<ReturnType<typeof validateSession>>;';
+						'export type SessionValidationResult = Awaited<ReturnType<typeof validateSessionToken>>;';
 					ms.append(`\n\n${sessionType}`);
 				}
-
+				if (!ms.original.includes('async function invalidateSession')) {
+					const invalidateSession = dedent`					
+						${ts('', '/** @param {string} sessionId */')}
+						export async function invalidateSession(sessionId${ts(': string')})${ts(': Promise<void>')} {
+							await db.delete(table.session).where(eq(table.session.id, sessionId));
+						}`;
+					ms.append(`\n\n${invalidateSession}`);
+				}
+				if (!ms.original.includes('export function setSessionTokenCookie')) {
+					const setSessionTokenCookie = dedent`					
+						${ts('', '/**')}
+						${ts('', ' * @param {import("@sveltejs/kit").RequestEvent} event')}
+						${ts('', ' * @param {string} token')}
+						${ts('', ' * @param {Date} expiresAt')}
+						${ts('', ' */')}
+						export function setSessionTokenCookie(event${ts(': RequestEvent')}, token${ts(': string')}, expiresAt${ts(': Date')}) {
+							event.cookies.set(sessionCookieName, token, {
+								expires: expiresAt,
+								path: '/'
+							});
+						}`;
+					ms.append(`\n\n${setSessionTokenCookie}`);
+				}
+				if (!ms.original.includes('export function deleteSessionTokenCookie')) {
+					const deleteSessionTokenCookie = dedent`					
+						${ts('', '/** @param {import("@sveltejs/kit").RequestEvent} event */')}
+						export function deleteSessionTokenCookie(event${ts(': RequestEvent')}, token${ts(': string')}, expiresAt${ts(': Date')}) {
+							event.cookies.set(sessionCookieName, '', {
+								expires: expiresAt,
+								path: '/'
+							});
+						}`;
```

Comment:
any particular reason to use `event.cookies.set` over `event.cookies.delete`?
```suggestion
						export function deleteSessionTokenCookie(event${ts(': RequestEvent')}, expiresAt${ts(': Date')}) {
							event.cookies.delete(sessionCookieName, { path: '/' });
						}`;
```

Reply:
Should be fixed now